README.rst

```diff
@@ -4,3 +4,15 @@ Thoth Messaging
 This provides a library called `thoth-messaging
 <https://pypi.org/project/thoth-messaging>`_ used in project `Thoth
 <https://thoth-station.ninja>`_.  It is a basic module to encapsule all messaging (here it is Kafka via Faust) primitives.
+
+Message Factory
+###############
+For the purposes of `Thoth` we have a few of our custom messages defined within the messaging module.  However, we
+wanted to allow individuals to be able to use our module without having to add their own messages first.  For testing
+and development purposes you can use `message_factory(...)` which allows you to create arbitrary messages using
+`thoth-messaging` using the following syntax:
```

Comment:
Isn't it the de facto way to create messages from now on if we have this generalization? All messages created until now should be removed actually and make sure we use the `message_factory` everywhere in our repos :) WDYT @KPostOffice?

Reply:
Hmmm, I'm not sure. I kinda like having all of our messages in a central location but I understand why we'd get rid of it so the library becomes less "thoth specific"

Reply:
I think the idea behind having these messages as classes is similar to having classes in ORM world. One can easily verify the message content and keep messages consistent in the application.

Reply:
So we leave it as is then?

Reply:
IMHO it will require some maintenance and library versioning, managing releases. It might be beneficial long term.